pkg/math/math.go

```diff
@@ -19,6 +19,14 @@ func Min[V int | int64 | uint64 | int32 | uint32](vals ...V) V {
 	return minimum
 }
 
+// Abs returns the absolute value of a given number.
+func Abs[V int | int32 | int64](val V) V {
```

Comment:
we should use this predefined type constraint 

https://pkg.go.dev/golang.org/x/exp/constraints#Signed